src/test/java/net/masterthought/cucumber/generators/integrations/helpers/OutputAssertion.java

```diff
@@ -8,7 +8,7 @@
 public class OutputAssertion extends ReportAssertion {
 
     public void hasMessages(String[] messages) {
-        WebAssertion[] outputMessages = allBySelector("pre", WebAssertion.class);
+        WebAssertion[] outputMessages = allBySelector("span", WebAssertion.class);
```

Comment:
Earlier each message used to be in it's own <pre> block. Currently all messages are in <pre> block  within each step. This selector returns these messages